cmd/network/verification.go

```diff
@@ -464,6 +479,85 @@ func (e *EgressVerification) getSecurityGroupId(ctx context.Context) (string, er
 	return *resp.SecurityGroups[0].GroupId, nil
 }
 
+// getCABundle retrieves a cluster's CA Trust Bundle from Hive
+func (e *EgressVerification) getCABundle(ctx context.Context) (string, error) {
+	if e.cluster == nil || e.cluster.AdditionalTrustBundle() == "" {
+		// No CA Bundle to retrieve for this cluster
+		return "", nil
+	}
+
+	scheme := runtime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil {
+		return "", err
+	}
+	// Register hivev1 for SelectorSyncSets
+	if err := hivev1.AddToScheme(scheme); err != nil {
+		return "", err
+	}
+
+	hive, err := utils.GetHiveCluster(e.cluster.ID())
+	if err != nil {
+		return "", err
+	}
+
```

Comment:
**thought:** This is unfortunately now a difference in the effectivity of CAD running network verifier and manually running it. 

FYI @RaphaelBut

Reply:
AFAIK - we don't. Right now, CAD just runs the network verifier pretty much in the same way osdctl did up until this point. We won't be able to get the CA as we have no hive access, unless it's saved in OCM as well?